content/games.html

```diff
@@ -166,6 +175,11 @@
 <noscript><div class="noscript">Please enable JavaScript to view the server list.</div></noscript>
 <div id="servercontainer" style="display: none">
 <p><span id="activePlayers">0</span> players are online in <span id="activeServers">0</span> advertised servers.</p>
+<form name="show">
+  <b>Show:</b>
+  <input type="checkbox" name="started" onchange="updateGameList()" checked=true> already started
```

Comment:
These should also be wrapped in `<label>`s for accessibility (increasing the click target to the whole entry instead of just the box).